reoptjl/validators.py

```diff
@@ -663,5 +663,5 @@ def lat_lon_in_windtoolkit(lat, lon):
     y = int(round((point[1] - origin[1]) / 2000))
     y_max, x_max = (1602, 2976)
     if (x < 0) or (y < 0) or (x >= x_max) or (y >= y_max):
-        raise ValueError("Latitude/Longitude is outside of wind resource dataset bounds.")
+        return None
```

Comment:
By returning None the `validation_error` will be included in the output

Reply:
so the new behavior is to try and get wind resource data for any provided inputs. Sites which are truly beyond the international dataset will get a validation error from REopt?

Reply:
Sites beyond the international dataset **that also do not provide production_factor_series** will get an error (before all international sites did). See change to line 291